cmd/rsamtrig-config/main.go

```diff
@@ -0,0 +1,262 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/GeoNet/delta"
+	"github.com/GeoNet/delta/meta"
+)
+
+type Settings struct {
+	base     string  // optional delta base directory
+	level    float64 // default rsam level
+	output   string  // output caps key directory
+	stations string  // list of stations to exclude
```

Comment:
issue: I can't relate this comment to what is going on in the code

Reply:
agreed, this is a copy/paste error without updating the comment.